packages/@aws-cdk/aws-eks/lib/cluster.ts

```diff
@@ -959,14 +959,14 @@ export class EksOptimizedImage implements ec2.IMachineImage {
   /**
    * Constructs a new instance of the EcsOptimizedAmi class.
    */
-  public constructor(props: EksOptimizedImageProps) {
-    this.nodeType = props && props.nodeType;
-    this.kubernetesVersion = props && props.kubernetesVersion || LATEST_KUBERNETES_VERSION;
+  public constructor(props: EksOptimizedImageProps = { }) {
+    this.nodeType = props.nodeType ?? NodeType.STANDARD;
```

Comment:
@pahud I piggy backed on your PR to fix another issue related to `EksOptimizedImage` (#6891). Can I bug you to add a unit test for this?

Reply:
Sure! One more thing - as the default cluster creation now goes to 1.15 as the default k8s version. Are we going to change `LATEST_KUBERNETES_VERSION` to 1.15 or just keep it as is(1.14)?

https://github.com/aws/aws-cdk/blob/307c8b021d5c00c1d675f4ce3cba8004a6a4a0a8/packages/%40aws-cdk/aws-eks/lib/cluster.ts#L1018-L1019

Reply:
I was running the integ testing.

Now as the default version goes to `1.15`, the managed nodegroup AMI goes to 1.15

AMI ID **amazon-eks-node-1.15-v20200228 (ami-06abd5347585f6519)**

and the self-managed AMI will goes to `1.14` because of the `LATEST_KUBERNETES_VERSION`

AMI ID **amazon-eks-node-1.14-v20200423 (ami-0bc4f8babcbafed56)**

Reply:
I think we should not offer a default Kubernetes version and require that version always be explicitly set. The current behavior is not sustainable 

Let’s do it in a separate PR though.